styles/main.css

```diff
@@ -91,139 +91,76 @@ a {
     top: 5px;
 }
 
-#nav a {
-    color: #fff;
+a {
+    color: #897F7B;
 }
 
-#nav a:hover {
-    color: #D6E8D9;
+a:hover {
+    color: #E94E1B;
 }
     
 #about-page .active,
-#illustration-page .active,
+#work-page .active,
 #webdesign-page .active {
-    color: #D6E8D9;
+    color: #E94E1B;
 }
 
 
-
-.banner img {
-    width: 100%;
-    max-height: 15px;
-}
-
-#content {
-    width: 980px;
+.content {
     margin: auto;
     overflow: hidden;
     clear: both;
 }
 
 #footer {
-    max-height: 15px;
     width: 100%;
     position: absolute;
     left: 0;
     bottom: 0;
 }
 
-/* SPLASH --------------------------------*/
-
-.radius {
-    border-radius: 50%;
-    width: 250px;
-    height: 250px;
-    background-color: #f2f2f2;
-    border: 1px solid #ccc;
-}
-
-
-#splash-page {
-    background: url("../images/splashbg.png") repeat fixed;
-}
-
-#splash-page #content2 {
-    margin: 140px auto 0;
-    width: 900px;
-    text-align: center;
-}
-
-#splash-logo {
-    display: block;
-    width: 300px;
-    height: 100px;
-    margin: auto;
-}
-
-#splash-logo .splash-text {
-    padding-top: 10px;
-    line-height: 25px;
-    font-size: 20px;
-    width: 300px;
-}
-
-
-#splash-buttons {
-    left: 5px;
-    display: inline-block;
-    margin: 65px;  
-}
-
-#splash-buttons:hover {
-    opacity: 0.7;
-}
-
-
-.splash-text {
-    font-variant: small-caps;
-    font-size: 20px;
+.copyright p {
+    font-size: 10px;
+    color: #897F7B;
     text-align: center;
-    line-height: 250px;
-    font-family: 'Montserrat', sans-serif;
-    color: #86a59c;
-    display: block;
+    margin: 50px auto 25px auto;
 }
 
 /* GALLERY --------------------------------*/
 
 #card {
-    width: 230px;
-    height: 285px;
+    width: 310px;
+    height: 234px;
     background: #fff;
-    border: 1px solid #cad6cb;
     float:left;
-    margin: 15px 6px;
+    margin: 15px 8px;
 }
 
 #card:hover {
     opacity: 0.8;
-    background-color: #F4F5F4;
-}
-
-#thumb {
-    width: 220px;
-    height: 230px;
-    margin: 5px;
 }
 
 #title_card {
-    margin: 10px;
+    margin-top: 10px;
 }
 
 #title_card .title {
-    font-weight: 700;
-    font-size: 14px;   
+    color: #897F7B;
+    font-size: 13px;   
 }
 
-#title_card .sub-title {
-    font-size: 10px;
-    color: #86a59c;   
-    line-height: 2;
+#title_card .worktype {
+    color: #E94E1B;
+    font-size: 12px;   
 }
 
 
 /* ABOUT PAGE -----------------------------*/
 
+#about-page #gallery h1{
```

Comment:
h1 on about page is different. Other internal pages have a margin-left:5px, but not on the about page. this one is specifying so that it's aligned to the body text.